apps/src/authoredHints.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * @overview helper class to manage the state of the Authored Hint UI.
+ * Used exclusively by StudioApp.
+ */
+
+var dom = require('./dom');
+var HintSelect = require('./templates/hintSelect.jsx');
+var authoredHintUtils = require('./authoredHintUtils');
+var lightbulbSVG = require('./templates/lightbulb.svg.ejs')();
+var lightbulbDimSVG = require('./templates/lightbulb_dim.svg.ejs')();
```

Comment:
I am entirely unsure how I want to go about including these SVG elements. The only thing I am sure of is that there has to be a better way than this.